src/routes/LogInForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import users from "../localDB/users.json";
+import { useNavigate } from "react-router-dom";
+import "./login.css";
+
+export default function LogInForm() {
+  const [usersData, setUsersData] = React.useState(users);
+  const navigate = useNavigate();
+
+  const [inputPassword, setInputPassword] = React.useState("");
+  const [inputEmail, setInputEmail] = React.useState("");
+
+  let loggedIn = false;
+
+  function login() {
+    navigate("/main");
+  }
+
+  function validateLogin() {
+    usersData.map(function ({ id, email, password }) {
+      if (email == inputEmail && password == inputPassword) {
```

Comment:
I will do all this comparing using `===` instead of `==` https://www.scaler.com/topics/javascript/difference-between-double-equals-and-triple-equals-in-javascript/